sphinx/source/docs/user_guide/annotations.rst

```diff
@@ -225,7 +225,7 @@ Box Annotations
 
 A |BoxAnnotation| can be linked to either data or screen coordinates in order
 to emphasize specific plot regions. By default, box annotation dimensions (e.g.
-``left`` or ``top``) default will extend the annotation to the edge of the
+``left`` or ``top``) will extend the annotation to the edge of the
 plot area.
```

Comment:
nit: new line before "plot area" could now be removed

Reply:
Thanks, I'll include this in my next commit!